toolchain/check/eval.cpp

```diff
@@ -1122,6 +1123,12 @@ auto TryEvalInst(Context& context, SemIR::InstId inst_id, SemIR::Inst inst)
           Phase::Template);
     }
 
+    case CARBON_KIND(SemIR::SpecificConstant instance): {
+      // Pull the instance-specific constant value out of the generic instance.
```

Comment:
Note, comments like this are part of why I was asking my other question... To me, this is now just producing out the specific's constant value. "instance-specific" reads a little awkward for me if we're trying to use "specific" as a term.